assets/sass/imports/_colors.scss

```diff
@@ -0,0 +1,169 @@
+            sch-blue1: #3EB5D9,
+            sch-blue2: #395D6F,
+            sch-green: #3AB298,
+            sch-orange: #F99532,
+            sch-accent: rgb(108, 108, 108),
+            sch-accent0: rgb(127, 127, 127),
+            sch-accent1: rgb(205, 205, 205),
+            sch-accent2: rgb(228, 228, 228),
+            sch-accent3: rgb(233, 233, 233),
+            sch-accent4: rgb(252, 252, 252)
+        ),
+        $dark-theme: (
+            background-color: rgb(20, 20, 20),
+            label: rgb(0, 0, 0),
+            shadow: rgba(255, 255, 255, 0.26),
+            form-border-color: rgb(104, 104, 104),
+
+            apl-red: #FF3B30,
+            apl-blue: #0A84FF,
+            apl-green: #30D158,
+            apl-yellow: #FFD60A,
+            apl-gray: #8E8E93,
+            apl-red2: #FF453A,
+            apl-background: #FFFFFFFF,
+            apl-background2: #EBEBF599,
+            apl-background3: #EBEBF54D,
+            apl-purple: #BF5AF2,
+
+            sch-purple: rgb(141, 139, 193),
+            sch-purple2: rgb(106, 101, 164),
+            sch-purple3: rgb(65, 53, 153),
+            sch-red1: #E64D43,
+            sch-red2: #EF5349,
+            sch-blue1: #5b8296, //lighter
+            sch-blue2: #3EB5D9,
+            sch-green: #3AB298,
+            sch-orange: #F99532,
+            sch-accent: rgb(147, 147, 147),
+            sch-accent0: rgb(127, 127, 127),
+            sch-accent1: rgb(50, 50, 50),
+            sch-accent2: rgb(37, 37, 37),
+            sch-accent3: rgb(27, 27, 27),
+            sch-accent4: rgb(10, 10, 10)
+        )
+);
+
+//converting scss variable into css variable
+:root {
+    --theme: $light-theme;
+    $light: map-get($themes, $light-theme);
+    @each $name, $color in $light {
+        --color-#{$name} : #{$color};
+        --color-#{$name}-r: #{red($color)};
+        --color-#{$name}-g: #{green($color)};
+        --color-#{$name}-b: #{blue($color)};
+        --color-#{$name}-a: #{alpha($color)};
+        --color-#{$name}-h: #{hue($color)};
+        --color-#{$name}-s: #{saturation($color)};
+        --color-#{$name}-l: #{lightness($color)};
+        
+    }
+}
+
+
+.dark-mode {
+    --theme : $dark-theme;
+    $dark: map-get($themes, $dark-theme);
+    @each $name, $color in $dark {
+        --color-#{$name} : #{$color};
+        --color-#{$name}-r: #{red($color)};
+        --color-#{$name}-g: #{green($color)};
+        --color-#{$name}-b: #{blue($color)};
+        --color-#{$name}-a: #{alpha($color)};
+        --color-#{$name}-h: #{hue($color)};
+        --color-#{$name}-s: #{saturation($color)};
+        --color-#{$name}-l: #{lightness($color)};
+    }
+}
+
+//creating a function to get color instead of using css varible
+@function getColor($color-name) {
+    @return var(--color-#{$color-name});
+}
+
+//change brigthness / lightness of color *by* $amount (+ lighter, - darker)
+//$amount should be in percent 0% - 100%
+@function lightenColor($colorName, $amount) {
```

Comment:
not the best name I ever came up with, but I think it's understandable. Open for suggestions..